src/create/cli.js

```diff
@@ -6,6 +6,10 @@ generate({
   arcFile: path.join(process.cwd(), '.arc'),
   execute: true
 },
-function() {
-  console.log('\n')
+function(err) {
+  if (err) {
+    console.error('Error creating resources: ', err)
```

Comment:
Bonus change: Errors weren't getting logged out if the CLI failed for some reason.